Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -1393,55 +1520,114 @@ function vtkOpenGLTexture(publicAPI, model) {
     model._openGLRenderWindow.activateTexture(publicAPI);
     publicAPI.createTexture();
     publicAPI.bind();
+
+    const hasUpdatedExtents = updatedExtents.length > 0;
+    const paramCache = model._paramCache;
+
+    // It's possible for the texture parameters to change while
+    // streaming, so check for such a change.
+    const rebuildEntireTexture =
```

Comment:
You can create a `newParamCache` object and call `DeepEqual` between this `newParamCache` and `model._paramCache`. Then, if they are different, you just have to do `model._paramCache = newParamCache;`